database-plugins/src/main/java/io/cdap/plugin/db/batch/sink/DBSink.java

```diff
@@ -115,6 +117,9 @@ public void prepareRun(BatchSinkContext context) {
       DBUtils.cleanup(driverClass);
     }
     context.addOutput(Output.of(dbSinkConfig.referenceName, new DBOutputFormatProvider(dbSinkConfig, driverClass)));
+
+    recordLineage(context, dbSinkConfig.referenceName, Schema.of(Schema.Type.STRING),
```

Comment:
Why do you construct a new Schema here? We should NEVER construct the schema. You can get the schema using `context.getInputSchema()`. And then fields can be derived like https://github.com/cdapio/hydrator-plugins/blob/develop/format-common/src/main/java/io/cdap/plugin/format/plugin/AbstractFileSink.java#L96. Please do take a look what the context can give you since that is essential about the fll